src/playlist/playlistview.h

```diff
@@ -244,6 +246,8 @@ class PlaylistView : public QTreeView {
   int drop_indicator_row_;
   bool drag_over_;
 
+  bool rating_lock_status_; // To store Ratings section lock status
```

Comment:
Can you rename this to something more obvious about which way round is locked or not? e.g. `ratings_locked_`